BigCommerce/Integration/04_ManageProductData.md

```diff
@@ -0,0 +1,99 @@
+# Manage product and offer data
+
+Before you start exchanging data with *BigCommerce*, you must check your product and offer data.
+You must check the images assigned to an offer in the *PIM* or the *Omni-Channel* module. In addition, you can define the color-swatch variant option.
+
+
+## Check product images
+
+Verify that the desired product data image is in the correct position and that the correct text field is maintained.   
+
+It is recommended for a good presentation of the offers in your store that the desired images are displayed for each variant. *BigCommerce* supports the import of one thumbnail per variant only. The first image in the corresponding attribute is marked as thumbnail image for the offer. Additionally, the alt-text of that image will be added as description.   
```

Comment:
Vorschlag: For a good presentation of the offers in your store, it is recommended that the desired images are displayed for each variant.

Reply:
Done